stock_secondary_unit/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2018 Tecnativa - Sergio Teruel
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Stock Secondary Unit",
+    "summary": "Get product quantities in a secondary unit",
+    "version": "13.0.1.0.0",
+    "development_status": "Beta",
```

Comment:
Change status to "Production/Stable" and add maintainers key

Reply:
Edited.